research/fedbn/jobs/fed_bn/app/config/config_fed_client.conf

```diff
@@ -0,0 +1,77 @@
+{
+  format_version = 2
+  app_script = "fedbn_cifar10.py"
+  app_config = ""
```

Comment:
suggest to not include this generated job folder "fed_bn" in this PR, since users can just use your "create_job.sh" to generate this exact same job.

Reply:
ok got it